spec/stack_master/parameter_resolver_spec.rb

```diff
@@ -55,4 +63,29 @@ def resolve(params)
       expect(resolve(param: { my_resolver: 2 }, param2: { my_resolver: 2 })).to eq(param: 10, param2: 10)
     end
   end
+
+  context 'when the resolver class already exist' do
+    it 'does not try to load it' do
+      expect(subject).to receive(:load_parameter_resolver).once.and_call_original
+      expect(subject).not_to receive(:require_parameter_resolver)
+
+      subject.resolve
```

Comment:
This would read better with named subjects - `subject(:parameter_resolver) { .. }`